src/CodeCoverage.php

```diff
@@ -374,10 +395,11 @@ public function merge(self $that): void
             \array_merge($this->filter->getWhitelistedFiles(), $that->filter()->getWhitelistedFiles())
         );
 
-        foreach ($that->data as $file => $lines) {
+        // I don't know how / why this works, but it should be refactored to ->getData()
```

Comment:
PHP is crazy and sets visibility based on class, not instance. `private` and `protected` members are visible between instances of the same class.

![crazy, right?](https://user-images.githubusercontent.com/691492/54761849-4767b780-4bc9-11e9-9bd9-10752aa10a8b.gif)